collectd-docker.py

```diff
@@ -222,6 +222,10 @@
     'docker-librato.\w+.cpu_stats.cpu_usage.percpu_usage.*',
 ]
 
+whitelist = [re.compile(l) for l in WHITELIST_STATS]
+blacklist = [re.compile(l) for l in BLACKLIST_STATS]
```

Comment:
I'm guessing compiling the regular expressions was not actually a performance bottleneck, but this makes sense regardless.